### PR TITLE
Include self-reference in hreflang tags

### PR DESCRIPTION
**Pull request by psolbach**
In compliance with Google specification, we add an additional hreflang tag for the language currently rendered. See https://developers.google.com/search/docs/specialty/international/localized-versions?hl=de

### CODE REVIEW

File: hugo-modules/core/utils/seo/private/get-tags.html [hunk 1]
```diff
@@ -42,16 +40,22 @@
 {{ with $seo.locale }}
   {{ $c := dict "key" "locale" "value" .}}
   {{ $tags = $tags | append (partialCached "utils/seo/tag/og" $c $c) }}
+
+  {{ $attr := dict "rel" "alternate" "href" $.Permalink "hreflang" . }}
+  {{ $c2 := dict "name" "link" "attr" $attr }}
+  {{ $tags = $tags | append $c2 }}
 {{ end }}
 
+{{ warnf $.Permalink }}
```
Comment (peterschewe) [inline hugo-modules/core/utils/seo/private/get-tags.html:49]: Was this just for debugging or do we need it for logging in production?

Comment (psolbach) [reply hugo-modules/core/utils/seo/private/get-tags.html:49]: Ah, yes, L49 should go

Comment (psolbach) [reply hugo-modules/core/utils/seo/private/get-tags.html]: And gone

**Review by peterschewe — COMMENTED** [review]
Works as aspected. Should we also add `x-default`?

**Review by peterschewe — APPROVED** [review]

**Comment by psolbach** [comment]
> Works as aspected. Should we also add `x-default`?

I believe the default should be explicitly defined in Contentful or elsewhere. I'll create another ticket.

**Pull request merged**